package.json

```diff
@@ -151,6 +152,7 @@
     "globalthis": "^1.0.2",
     "hammerjs": "^2.0.8",
     "hash-wasm": "^4.4.1",
+    "howler": "^2.2.1",
```

Comment:
How big (minified & gzipped) is this library? Is there any possible alternative?

Reply:
7kb gzipped. 9.3kb minified. We can try using HTML audio but I feel that it could become a real pain to maintain that code. Howler is a really powerful js library and it actually has not dependencies.

Reply:
Okay, that seems reasonable.